table_evaluator/TEST_jac_sim.ipynb

```diff
@@ -0,0 +1,68 @@
+{
+ "cells": [
```

Comment:
Make sure that adding the notebook adds value. Currently, I think it just shows an error, which is not adding that much. So either put some other things here or remove the notebook.

Reply:
I will remove this one.